ui/l10n/en.js

```diff
@@ -554,6 +554,10 @@ var dictionary = {
 "label.compute":"Compute",
 "label.compute.and.storage":"Compute and Storage",
 "label.compute.offering":"Compute offering",
+"label.compute.offering.type":"Compute offering type",
+"label.compute.offering.custom.constrained":"Custom Constrained",
+"label.compute.offering.custom.unconstrained":"Custom Unsconstrained",
```

Comment:
```suggestion
"label.compute.offering.custom.unconstrained":"Custom Unconstrained",
```

Reply:
Good hack. Simply the ui typing field could be grey out but that won't solve the api call. This need a service layer check imho.